docs/root/intro/arch_overview/cluster_manager.rst

```diff
@@ -24,3 +24,17 @@ distribution.
 
 * Cluster manager :ref:`configuration <config_cluster_manager>`.
 * CDS :ref:`configuration <config_cluster_manager_cds>`.
+
+Cluster warming
+---------------
+
+When clusters are initialized both at server boot as well as via CDS, they are "warmed." This means
+that clusters do not become available until the following operations have taken place.
```

Comment:
It is not immediately clear what availability implies.. Does cluster unavailability imply that:
- routes that reference the cluster fail `validate_clusters` check, and
- listeners that reference the cluster remain in the warming state?